app/helpers/publishers_helper.rb

```diff
@@ -111,13 +111,15 @@ def terms_of_service_url
     Rails.application.secrets[:terms_of_service_url]
   end
 
-  def publisher_default_currency(publisher)
-    publisher.default_currency.present? ? publisher.default_currency : 'BAT'
-  end
-
   def publisher_available_currencies(publisher)
     available_currencies = publisher.wallet.try(:wallet_details).try(:[], 'availableCurrencies')
-    available_currencies.blank? ? ['BAT'] : available_currencies
+    if available_currencies.blank?
+      available_currencies = ['BAT']
```

Comment:
@nvonpentz I think this default may cause more problems than it solves, now that I understand that not all publishers can choose `BAT`. It's probably better to leave the options blank and have `default_currency` remain `nil` until publishers can choose from actual available currencies pulled from eyeshade.